src/commands/mapsheet-coverage/mapsheet.coverage.ts

```diff
@@ -0,0 +1,224 @@
+    }),
+    location: option({
+      type: string,
+      long: 'location',
+      description: 'Location of the basemaps configuration file',
+      defaultValueIsSerializable: true,
+      defaultValue: () => 'https://raw.githubusercontent.com/linz/basemaps-config/master/config/tileset/elevation.json',
+    }),
+    mapSheet: option({
+      type: optional(string),
+      long: 'mapsheet',
+      description: 'Limit the output to a specific mapsheet eg "BX01"',
+    }),
+  },
+  async handler(args) {
+    const startTime = performance.now();
+    registerCli(this, args);
+    logger.info('MapSheet:Start');
+
+    if (!ValidCodes.has(args.epsgCode)) {
+      logger.error({ epsgCode: args.epsgCode }, 'Invalid:EpsgCode');
+      return;
+    }
+
+    const config = await fsa.readJson<ConfigTileSetRaster>(args.location);
+
+    /** All the layer's capture areas areas with some additonal metadata */
+    const allLayers = { type: 'FeatureCollection', features: [] as GeoJSON.Feature[] };
+
+    // All previous capture area features restricted to the area needed for the output
+    const layersRequired = { type: 'FeatureCollection', features: [] as GeoJSON.Feature[] };
+
+    // All the coordinates currently used
+    let layersCombined: pc.MultiPolygon[] = [];
+
+    // MapSheetName to List of source files required
+    const mapSheets = new Map<string, string[]>();
+
+    // Reverse the configuration so the highest priority datasets come first
+    for (const layer of config.layers.reverse()) {
+      if (Skip.has(layer.name)) continue;
+
+      const layerSource = layer[args.epsgCode as 2193 | 3857];
+      if (layerSource == null) {
+        logger.warn({ layer: layer.name, layerSource: args.epsgCode }, 'Layer:Missing');
+        continue;
+      }
+
+      const targetCollection = new URL('collection.json', layerSource);
+      const collection = await fsa.readJson<StacCollection>(targetCollection.href);
+
+      // Capture area is the area where this layer has data for
+      const captureAreaLink = collection.assets?.['capture_area'];
+      if (captureAreaLink == null) throw new Error('Missing capture area :' + targetCollection.href);
+      const targetCaptureAreaUrl = new URL(captureAreaLink.href, targetCollection.href);
+
+      const captureArea = await fsa.readJson<GeoJSON.Feature>(targetCaptureAreaUrl.href);
+      // Propagate properties from the source STAC collection into the capture area geojson
+      captureArea.properties = captureArea.properties ?? {};
+      captureArea.properties['source'] = targetCaptureAreaUrl.href;
+      captureArea.properties['title'] = collection.title;
+      for (const [key, value] of Object.entries(collection)) {
+        if (key.startsWith('linz:')) captureArea.properties[key] = value;
+      }
+      logger.debug(
+        {
+          layer: layer.name,
+          title: collection.title,
+          layerSizeBytes: Math.round(JSON.stringify(captureArea.geometry).length),
```

Comment:
Is this meant as a rough approximation of the size?

Also, is `Math.round` necessary? The length should always be an integer.

Reply:
It is a relic of when this was layerSizeKb and can be removed